queries/tsx/highlights.scm

```diff
@@ -0,0 +1 @@
+; inherits: typescript,jsx
```

Comment:
~~and javascript. also the order was fine-tuned in some way, I can't recall~~

Reply:
Exactly